test/models/site_test.rb

```diff
@@ -20,4 +26,32 @@ class SiteTest < ActiveSupport::TestCase
     assert_equal(0, Post.count)
     assert_equal(0, Image.count)
   end
+
+  sub_test_case 'name' do
+    test 'null' do
+      site = Site.create(js_url: "http://example.com/application.js",
+                         css_url: "http://example.com/application.css")
+      assert_false(site.valid?)
+      assert_equal("can't be blank", site.errors[:name][0])
```

Comment:
Thanks! We can use `assert_includes` but I think it should be failed if site has some other errors that are unexpected. However, we should get some other errors easy. I'll compare remain array.